src/coreComponents/constitutive/fluid/PVTFunctions/FenghourCO2ViscosityFunction.cpp

```diff
@@ -96,15 +96,15 @@ void FenghourCO2ViscosityFunction::MakeTable( string_array const & inputPara )
   real64_array2d viscosities( nP, nT );
   real64_array2d densities( nP, nT );
 
-  SpanWagnerCO2DensityFunction::CalculateCO2Density( pressures, temperatures, densities );
+  SpanWagnerCO2DensityFunction::calculateCO2Density( pressures, temperatures, densities );
```

Comment:
I am guessing you had to hand correct `CO2` from `Co2`?

Reply:
Actually I did not think I did 🤷‍♂️